assignments/git-1-R.md

```diff
@@ -5,4 +5,15 @@ language: R
 exercises: ['Version Control 1', 'Version Control 2', 'Version Control 3', 'Version Control 4', 'Version Control 5', 'Version Control 6']
 ---
 
+####Learning Objectives
+
+> Following this assignment students should be able to:
+
+> - establish a GitHub local repository
```

Comment:
If it's local it isn't GitHub. Regardless I like keeping this big picture so "establish a local version control repository"